bigtop-deploy/juju/hadoop-mapreduce/README.md

```diff
@@ -50,8 +50,13 @@ units. To deploy this bundle, simply use:
 
     juju quickstart bigtop-processing-mapreduce
 
```

Comment:
I think it's important for the directory structure here to match the bundle name in the store, so that the association is clear.  And I think it's important for the store to include at least "bigtop" and "yarn" (or "mapreduce"), so my preference is to make it "bigtop-processing-yarn" even though it's a bit longer.